addons/payment_asiapay/data/payment_acquirer_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="payment_acquirer_asiapay" model="payment.acquirer">
+            <field name="name">Asiapay</field>
+            <field name="provider">asiapay</field>
+            <field name="company_id" ref="base.main_company" />
+            <field name="display_as">Credit Card (powered by Asiapay)</field>
+            <field name="image_128" type="base64" file="payment_asiapay/static/src/img/asiapay_logo.png" />
+            <field name="redirect_form_view_id" ref="redirect_form" />
+            <field name="support_authorization">False</field>
+            <field name="support_fees_computation">False</field>
+            <field name="support_refund" />
+            <field name="support_tokenization">False</field>
```

Comment:
The base acquirer record and most of its fields (see other acquirers' implementation) are defined in the `payment` module's data to allow the acquirer to be discovered from the Kanban view without installing the related module.